src/obofoundry/utils.py

```diff
@@ -41,3 +42,20 @@ def query_wikidata(query: str):
     res.raise_for_status()
     res_json = res.json()
     return res_json["results"]["bindings"]
+
+
+def get_new_data():
+    """Get records for ontologies that have additional checks
```

Comment:
the Flake8 test wants you to put a period after "checks," can you please do that?

Reply:
addressed!